step.yml

```diff
@@ -1,9 +1,23 @@
-title: |-
-  apt-get install
-summary: |
-  Install or upgrade packages with apt-get
-description: |
-  Install or upgrade packages with apt-get
+title: "apt-get install"
+summary: |-
+  Install or upgrade packages with apt-get.
```

Comment:
We should mention that `apt-get` is a tool only available on Ubuntu, so this whole step should only be run on our Ubuntu stacks.
I would write something like `Install or upgrade packes on Ubuntu with apt-get`

Reply:
Install or upgrade packages on Ubuntu with `apt-get`.